Mage.Sets/src/mage/cards/h/HexgoldSlash.java

```diff
@@ -0,0 +1,70 @@
+package mage.cards.h;
+
+import mage.abilities.Ability;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.keyword.ToxicAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Outcome;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.target.common.TargetCreaturePermanent;
+
+import java.util.UUID;
+
+/**
+ *
+ * @author AhmadYProjects
+ */
+public final class HexgoldSlash extends CardImpl {
+
+    public HexgoldSlash(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.INSTANT}, "{R}");
+        
+
+        // Hexgold Slash deals 2 damage to target creature. If that creature has toxic, Hexgold Slash deals 4 damage to that creature instead.
+        this.getSpellAbility().addTarget(new TargetCreaturePermanent());
+        this.getSpellAbility().addEffect(new HexgoldSlashEffect());
+    }
+
+    private HexgoldSlash(final HexgoldSlash card) {
+        super(card);
+    }
+
+    @Override
+    public HexgoldSlash copy() {
+        return new HexgoldSlash(this);
+    }
+}
+
+class HexgoldSlashEffect extends OneShotEffect{
+    public HexgoldSlashEffect(){
+        super(Outcome.Damage);
+        staticText = "{this} deals 2 damage to target creature. If that creature has toxic, {this} deals 4 damage instead}";
```

Comment:
`}` at the end of the string

Reply:
> try to write descriptive commit messages

Ok I will try to do that from here on out. all requested changes have been implemented in the latest commit

Reply:
thanks, looks good to me